packages/browser-bundle/package.json

```diff
@@ -19,8 +19,7 @@
     "browserify": "16.2.3",
     "tinyify": "2.5.0"
   },
-  "dependencies": {
-  },
+  "dependencies": {},
```

Comment:
Please revert this change since this does not belong to this PR.

Reply:
sure, done